src/experiences.ts

```diff
@@ -10,20 +10,20 @@ export enum API {
     MongoDB = 'MongoDB',
     Graph = 'Graph',
     Table = 'Table',
-    DocumentDB = 'DocumentDB'
+    Core = 'Core'
 }
 
 export enum DBAccountKind {
     MongoDB = 'MongoDB',
-    GlobalDocumentDB = 'GlobalDocumentDB'
+    GlobalCore = 'GlobalCore'
```

Comment:
Kind is used when creating a cosmos db account here: https://github.com/microsoft/vscode-cosmosdb/blob/master/src/tree/CosmosDBAccountWizard/CosmosDBAccountCreateStep.ts#L24

Most of your changes are just for labels, but this one likely affects functionality. Are we sure Azure supports "GlobalCore" as the kind when creating an account?